bin/node-template/node/src/service.rs

```diff
@@ -18,6 +18,11 @@ native_executor_instance!(
 	node_template_runtime::native_version,
 );
 
+
+// TODO:: Please change this to crate, lrpc=localrpc
+#[path = "rpc.rs"]
```

Comment:
??

Reply:
I wasn't sure whether or not it was ideal to have the whole rpc.rs loaded into service.rs. If I recall correctly the original, in the node- crate::... is used, which would error out for me. Perhaps more reading on namespaces in Rust for me.

Reply:
Cant we treat `rpc.rs` as a traditional rust module? I am wondering why this attribute is needed